eng/ReferenceAssemblies.props

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup>
```

Comment:
nit its a little unusual to put this in the referenceassemblies.props file.

maybe a "conventions.props" or just put them in root directory.build.props?

EG:
```
<PropertyGroup>
  <IsPartialFacadeAssembly Condition="'$(IsPartialFacadeAssembly)'=='' AND ($(MSBuildProjectName.EndsWith('.Facade')))">true</IsPartialFacadeAssembly>
  <IsReferenceAssembly Condition="'$(IsReferenceAssembly)'=='' AND ($(MSBuildProjectName.EndsWith('.Ref')))">true</IsReferenceAssembly>
</PropertyGroup>
```

Reply:
I agree, I have some other props files that also need re-visiting, was going to move it somewhere more appropriate then, but the root directory.build.props sounds like a better choice for now.